EIPS/eip-2929.md

```diff
@@ -0,0 +1,165 @@
+---
+eip: 2929
+title: Gas cost increases for state access opcodes
+author: Vitalik Buterin (@vbuterin), Martin Swende (@holiman)
+discussions-to: https://ethereum-magicians.org/t/eip-2929-gas-cost-increases-for-state-access-opcodes/4558
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-09-01
+---
+
+## Simple Summary
+
+Increase the gas cost of SLOAD to 2100, and the CALL opcode family, BALANCE and the EXT* opcode family to 2600. Exempts (i) precompiles, and (ii) addresses and storage slots that have already been accessed in the same transaction. Additionally reforms SSTORE metering and SELFDESTRUCT to ensure "de-facto storage loads" inherent in those opcodes are priced correctly.
```

Comment:
```suggestion
Increases gas cost for `SLOAD`, `*CALL`, `BALANCE`, `EXT*` for the first time in a transaction.

## Abstract

Increase the gas cost of `SLOAD` (`0x54`) to 2100, and the `CALL` opcode family (`0xf1`, `f2`, `f4`, `fA`), `BALANCE` `0x31` and the `EXT*` opcode family (`0x3b`, `0x3c`, `0x3f`) to 2600. Exempts (i) precompiles, and (ii) addresses and storage slots that have already been accessed in the same transaction. Additionally reforms `SSTORE` metering and `SELFDESTRUCT` to ensure "de-facto storage loads" inherent in those opcodes are priced correctly.
```
Simple Summary should be incredibly terse.  Think of it like subtext under the title of a forum post or something.  Abstract on the other hand should basically be a human friendly version of the whole specification.  The abstract doesn't need to go into *all* of the details necessary for implementation, but someone reading the abstract should walk away with a pretty good high level understanding of the changes being proposed.

In this case, you had the simple summary which was a bit too long, and no abstract, so the suggestion above just basically splits it in two.